apps/drec-ui/src/apps/user/logic/admin/useAdminUpdateOrganizationFormLogic.ts

```diff
@@ -18,24 +18,24 @@ export type TAdminUpdateOrganizationFormValues = {
 };
 
 export const useAdminUpdateOrganizationFormLogic = (
-    user: OrganizationDTO
+    organization: OrganizationDTO
 ): Omit<GenericFormProps<TAdminUpdateOrganizationFormValues>, 'submitHandler'> => {
     const initialFormData = {
-        name: user?.name,
-        address: user?.address,
-        zipCode: user?.zipCode,
-        city: user?.city,
-        country: user?.country,
-        businessType: user?.businessType,
-        tradeRegistryCompanyNumber: user?.tradeRegistryCompanyNumber,
-        vatNumber: user?.vatNumber,
-        status: user?.status
+        name: organization?.name,
+        address: organization?.address,
+        zipCode: organization?.zipCode,
```

Comment:
If it is optional you should provide an empty string as fallback `organization?.zipCode ?? ''` to avoid errors of component changing from being uncontrolled to controlled. Happens when `undefined` is supplied as initialValue